tests/QMLEngine/qml/PropertiesBasic.qml

```diff
@@ -0,0 +1,10 @@
+Item{
```

Comment:
If this is not to test parser behaviour here, could you use `Item {`, please?
If this is for parser tests — could you create a separate entry for them?